config/server-vars.yml

```diff
@@ -121,6 +121,7 @@ EDXAPP_RETRY_ACTIVATION_EMAIL_TIMEOUT: 2
 EDXAPP_SESSION_INACTIVITY_TIMEOUT_IN_SECONDS: 1800
 EDXAPP_X_FRAME_OPTIONS: "SAMEORIGIN"
 EDXAPP_CSRF_COOKIE_SECURE: %%NGINX_ENABLE_SSL%%
+NGINX_REDIRECT_TO_HTTPS: %%NGINX_ENABLE_SSL%% 
```

Comment:
make sure there's only one occurrence of NGINX_REDIRECT_TO_HTTPS in this file after you resolve any merge conflicts

Reply:
Will do.